client/cli/src/params/network_params.rs

```diff
@@ -33,11 +33,11 @@ use std::{borrow::Cow, path::PathBuf};
 #[derive(Debug, Clone, Args)]
 pub struct NetworkParams {
 	/// Specify a list of bootnodes.
-	#[clap(long, value_name = "ADDR", multiple_values(true))]
+	#[arg(long, value_name = "ADDR", num_args = 0..)]
```

Comment:
is `0..` ok here?

Reply:
Good catch, with `0..` it was possible to supply for example `--bootnodes` without address, which we dont want. Changed to `1..`.